blamo/lib/ObjectHandler.dart

```diff
@@ -519,13 +520,13 @@ class Test {
     }
     if(parsedJSON["percentRecovery"] != "null") {
       percentRecoveryConverted = parsedJSON["percentRecovery"].value;
-    }
+    }*/
 
     return Test(
       testType: parsedJSON["testType"],
-      beginTest: beginTestConverted,
```

Comment:
Something about this change caused a stack trace exception, where a double was expected but a string was recieved. Tests page wont populate now.
![image](https://user-images.githubusercontent.com/43582537/76709624-6bb38280-66bd-11ea-9228-5e320be07d1f.png)